app/experimenter/experiments/api/v5/mutations.py

```diff
@@ -68,8 +68,43 @@ def mutate(cls, root, info, input: ExperimentInput):
         return handle_with_serializer(cls, serializer)
 
 
+class EndExperiment(graphene.Mutation):
+    nimbus_experiment = graphene.Field(NimbusExperimentType)
+    message = ObjectField()
+    status = graphene.Int()
+
+    class Arguments:
+        input = ExperimentIdInput(required=True)
+
+    @classmethod
+    def mutate(cls, root, info, input: ExperimentIdInput):
+        experiment = NimbusExperiment.objects.get(id=input.id)
+
+        if experiment.status == NimbusExperiment.Status.LIVE:
```

Comment:
It's not a huge deal but I've learned that these kinds of status checks are better written as model properties because at some point we might add another status that's also part of what we consider LIVE and then there's only one place to update it rather than everywhere we check the status, but we could also leave that till later.

Reply:
We might want a follow up issue for that? Doesn't look like NimbusExperiment has model properties for status right now, and most spots I could see were just using constant comparisons like this

Reply:
Yeah we can file a followup 👍

Reply:
FWIW, we *do* have [a `getStatus` utility function](https://github.com/mozilla/experimenter/blob/main/app/experimenter/nimbus-ui/src/lib/experiment.ts#L9-L39) on the client side, but doesn't looks like we have the equivalent on the server side

Reply:
Oh interesting!